tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
```

Comment:
You also need `"allowJs": true` since all the code is still in JS files

Reply:
made the suggested change as part of commit 09be8e5fdff162327be808f6336dcd762cfd9d73